.github/workflows/rust.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  GST_PLUGIN_FEATURE_RANK: pulsesink:NONE,alsasink:NONE,jacksink:NONE,fakesink:MAX
```

Comment:
It's a good start but the envvar might grow with video sinks and so the test use them. 

overall, LGTM